CorsixTH/Lua/graphics.lua

```diff
@@ -336,10 +335,10 @@ function Graphics:onChangeLanguage()
   self.load_info = {} -- Any newly made objects are temporary, and shouldn't
```

Comment:
there is also a singular load_info variable there. Perhaps use load_info_table here instead?